### ISSUE TITLE
Needs Update 6.0

### ISSUE THREAD

**Issue by DiggityMan420** [Haplo064/ChatBubbles#22]
Needs to be updated to 6.0

**Comment by giosarm** [comment]
yeah i much prefer the bubbles than the chat box, makes it easier for me to follow conversations I'm in

**Comment by Sennarin** [comment]
It's been down for a couple of days only but I already miss this plugin greatly... Waiting for update.

**Comment by Judisch** [comment]
I've been testing the old version of this plugin (1.3.2.0) on Endwalker, and it seems to work with no issues. So I think zero code changes are required, just bump the `DalamudApiLevel` to 5. 

If you're a player, and you can't wait for the official update, then open up `C:\Users\Judisch\AppData\Roaming\XIVLauncher\installedPlugins\ChatBubbles\1.3.2.0\ChatBubbles.json` and change `"DalamudApiLevel": 4` to `"DalamudApiLevel": 5` to get this plugin working again.
